tests/unit/swagger/definitionsGeneration/metadata.spec.ts

```diff
@@ -946,4 +946,29 @@ describe('Metadata generation', () => {
       ]);
     });
   });
+
+  describe('controllerWithJsDocResponseDescriptionGeneration', () => {
+    const metadata = new MetadataGenerator('./fixtures/controllers/controllerWithJsDocResponseDescription.ts').Generate();
+    const controller = metadata.controllers[0];
+
+    it('has success response description', () => {
+      const description = 'SuccessResponse description';
+      const method = controller.methods[0]; // descriptionWithSuccessResponse
```

Comment:
Preferably, look these up using find, which is more descriptive/future-prof

Reply:
Sorry, I am afraid I do not understand what you mean.
Could you elaborate please? I would like to make it better if you think  there's something that can be improved :)